.ci/testruns/default/templates/testrun-default.yaml

```diff
@@ -359,8 +362,40 @@ spec:
       name: delete-shoot
       locationSet: upgraded
 
-  - name: delete-garden
+  - name: scheduler-test
     dependsOn: [ delete-shoot-aws, delete-shoot-gcp, delete-shoot-az, delete-shoot-gcp-upgraded, delete-shoot-aws-upgraded, delete-shoot-az-upgraded ]
+    definition:
+      name: scheduler-test
+      locationSet: upgraded
+      continueOnError: true
+      config:
+      - name: SHOOT_NAME
+        type: env
+        value: aws-scheduler
+      - name: CLOUDPROVIDER
+        type: env
+        value: aws
+      - name: K8S_VERSION
+        type: env
+        value: {{ .Values.shoot.k8sVersion }}
+      - name: SEED
+        type: env
+        value: gcp
+      - name: CLOUDPROFILE
+        type: env
+        value: aws
+      - name: SECRET_BINDING
+        type: env
+        value: core-aws-aws
```

Comment:
is this correct?

Reply:
@Diaphteiros this is the default naming convention of garden-setup?

Reply:
Yes, the naming scheme for the automatically created provider secrets/secretbindings is `<project>-<provider>-<seed/cloudprofile name>`, see [here](https://github.com/gardener/garden-setup/blob/master/components/gardencontent/coreproject/deployment.yaml#L23).